eng/pipelines/release-cli.yml

```diff
@@ -69,6 +69,8 @@ stages:
 
           - template: /eng/pipelines/templates/steps/install-terraform.yml
 
+          - template: /eng/pipelines/templates/steps/install-sysinternals.yml
```

Comment:
introduced some issues on CI on: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1886256&view=logs&j=557543dc-0cfb-5537-ec51-82b57469983e&s=68cb863b-765d-54b9-d65d-bc9485011ec8&t=87926c8c-91c3-5c8d-51cd-5e6f0d83746c&l=336

Reply:
temp fix/path: https://github.com/Azure/azure-dev/pull/784

@weikanglim to take a look later